contracts/test/CompoundGovernor.t.sol

```diff
@@ -418,6 +421,11 @@ contract HasVoted is CompoundGovernorTest {
 
         assertEq(governor.hasVoted(_proposalId, _voter), _hasVotedBefore ? true : false);
     }
+
+    function testFuzz_FalseReturnedIfHasVoted_Called_With_Invalid_ProposalId(uint256 _invalidProposalId, address _voter) public view {
```

Comment:
```suggestion
    function testFuzz_ReturnsFalseIfHasVotedCalledWithInvalidProposalId(uint256 _invalidProposalId, address _voter) public view {
```

Reply:
Done: [5d3add6](https://github.com/ScopeLift/compound-governance-upgrade/pull/54/commits/5d3add691daadd9c110819fb29fe14808ecc2b37)

Reply:
These still have underscores in the test names for some reason.

Reply:
https://github.com/ScopeLift/compound-governance-upgrade/pull/54/commits/6a318f6f7b18f5678887d096e193ac3403485148